containerz/containerz.proto

```diff
@@ -257,6 +343,50 @@ message StartContainerRequest {
   // List of volumes that should be attached to the container.
   repeated Volume volumes = 7;
 
+  // The network to attach this container to. This could be 'host', 'bridged',
+  // or any other network available in the runtime.
+  string network = 8;
+
+  message Capabilities {
+    repeated string add = 1;
+    repeated string remove = 2;
+  }
+  Capabilities cap = 9;
+
+  message Restart {
+    enum Policy {
+      // Never restart a container.
+      NONE = 0;
+
+      // Always restart a container unless explicitly stopped. This includes
+      // when the container runtime is restarted, i.e. any container with this
+      // policy prior to the runtime being stopped must be restarted when the
+      // runtime is started again.
+      ALWAYS = 1;
+
+      // Same as ALWAYS but not across container runtime restarts.
+      UNLESS_STOPPED = 2;
+
+      // Restart for max attempts if the container fails.
+      ON_FAILURE = 3;
+    }
+
+    Policy policy = 1;
+
+    // Only applies for the ON_FAILURE policy.
+    uint32 attempts = 2;
+  }
+  Restart restart = 10;
```

Comment:
Add documentation string for this field -- I presume this is set only as a container is started and tells us how to deal with restarting on failure of the container, or does it also apply to restart of the host system?

Reply:
Each policy enum has a description - I also added a generic description for the restart field.